server/repository/src/migrations/v2_02_00/sync.rs

```diff
@@ -0,0 +1,12 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+            ALTER TABLE sync_log ADD duration_in_seconds BIGINT DEFAULT 0 NOT NULL;
```

Comment:
yup - it was solely to avoid casting - as the `num_seconds()` call is returning `i64`
bit of a spurious reason though. I'll change to i32 and cast it.